cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("Database unavailable")
+                time.sleep(1)
+
+        self.stdout.write(self.style.SUCCESS("Database available!"))
```

Comment:
It's a good practice to return a success message when the database connection is established. However, it would be more informative if you could add more details to this message, such as the name of the database that is now available.